src/mscorlib/src/System/Runtime/CompilerServices/RuntimeWrappedException.cs

```diff
@@ -41,11 +41,6 @@ private RuntimeWrappedException(Object thrownObject)
             base.GetObjectData(info, context);
             info.AddValue("WrappedException", m_wrappedException, typeof(Object));
         }
-
-        internal RuntimeWrappedException(SerializationInfo info, StreamingContext context)
```

Comment:
Removing this will break deserialization, but I'm not sure if we ever actually throw these in CoreCLR.